fastapi_azure_auth/auth.py

```diff
@@ -328,3 +328,63 @@ def __init__(
             openapi_description=openapi_description,
         )
         self.scheme_name: str = 'Azure AD - PKCE, Multi-tenant'
+
+
+class B2CAuthorizationCodeBearer(AzureAuthorizationCodeBearerBase):
```

Comment:
Reading #87 it seems like there is a difference between single- and multi-tenant B2C setups. Should we rename this to `B2CMultiTenantAuthorizationCodeBearer` again? Sorry for going back on this..

Reply:
Sure